Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/RiskLevelResultExtensions.kt

```diff
@@ -17,6 +18,16 @@ fun List<EwRiskLevelResult>.tryLatestEwResultsWithDefaults(): DisplayableEwRiskR
     )
 }
 
+fun List<PtRiskLevelResult>.getLastCalculatedWithDefaults(): PtRiskLevelResult {
```

Comment:
What do you need this for? Defaults are provided in the RiskLevelStorage

Reply:
took it from the ew implementation but is now removed